fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1951,6 +1951,10 @@ public FragmentExecParams(PlanFragment fragment) {
                             params.getQuery_options().setBatch_size(SessionVariable.PIPELINE_BATCH_SIZE);
                         }
                         params.setPipeline_dop(fragment.getPipelineDop());
+
+                        boolean enableResourceGroup = ConnectContext.get().getSessionVariable().isEnableResourceGroup();
```

Comment:
need to check ConnectContext.get() whether  is null?

Reply:
@kangkaisen This is check in [L1943](https://github.com/StarRocks/starrocks/pull/3472/files#diff-8aadd0f1c5f7b1c677cfbeea3a3e39f4205e90037b199f4d3e822bc1984ab3beR1943).

```diff
                if (ConnectContext.get() != null) {
                    SessionVariable sessionVariable = ConnectContext.get().getSessionVariable();
                    if (isEnablePipelineEngine) {
                        boolean isPipeline =
                                fragment.getPlanRoot().canUsePipeLine() && fragment.getSink().canUsePipeLine();
                        params.setIs_pipeline(isPipeline);
                        if (isPipeline) {
                            params.getQuery_options().setBatch_size(SessionVariable.PIPELINE_BATCH_SIZE);
                        }
                        params.setPipeline_dop(fragment.getPipelineDop());

+                        boolean enableResourceGroup = ConnectContext.get().getSessionVariable().isEnableResourceGroup();
```